index.html

```diff
@@ -188,7 +192,7 @@ <h2 class="ton">Mult-Post<br>Stories</h2>
 
          </section>
 <!-- about myself -->
-         <section class="about">
+         <section class="about" id="about">
             <div class="about-me">
                 <h1>About Myself</h1>
```

Comment:
Good work implementing the hamburger button

Since we have to follow semantic tags with the HTML code. It would be nice to modify this line

- Kindly replace this `h1` tag with an `h2` or `h3` since for semantic rules there's the possibility to be only one h1 tag in your website